openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java

```diff
@@ -183,9 +183,12 @@ private String getTestSuiteCondition() {
 
   private String getFqnPrefixCondition(String tableName, String fqnPrefix) {
     return tableName == null
-        ? String.format("fqnHash LIKE '%s%s%%'", FullyQualifiedName.buildHash(fqnPrefix), Entity.SEPARATOR)
+        ? String.format(
+            "fqnHash LIKE '%s%s%%'",
```

Comment:
quoting a fqnPrefix is dangerous. This will cause lot of other things to break

Reply:
resolved